plugins/labhub.py

```diff
@@ -1,7 +1,258 @@
-from errbot import BotPlugin
+import json
+import os
+import re
 
+import github3
+from IGitt.GitHub.GitHub import GitHub
+from IGitt.GitLab.GitLab import GitLab
+from errbot import BotPlugin, re_botcmd
 
-class Labhub(BotPlugin):
-    """
-    GitHub and GitLab utilities
-    """
+from plugins import constants
+
+
+class LabHub(BotPlugin):
+    """GitHub and GitLab utilities"""  # Ignore QuotesBear
+
+    INVITE_SUCCESS = {
+        'newcomers': """
+ Welcome @{}! :tada:\n\nTo get started, please follow our
+ [newcomers guide](https://coala.io/newcomer). Most issues will be explained
+ there and in linked pages - it will save you a lot of time, just read it.
+ *Really.*\n\n*Do not take an issue if you don't understand it on your own.*
+ Especially if you are new you have to be aware that getting started with an
+ open source community is not trivial: you will have to work hard and most
+ likely become a better coder than you are now just as we all did.\n\n
+ Don't get us wrong: we are *very* glad to have you with us on this journey
+ into open source! We will also be there for you at all times to help you
+ with actual problems. :)
+""",
+        'developers': """
+ Wow @{}, you are a part of developers team now! :tada: Welcome to our
+ community!
+""",
+        'maintainers': """
+ @{} you seem to be awesome! You are now a maintainer! :tada: Please go
+ through https://github.com/coala/coala/wiki/Membership
+"""
+    }
+
+    GH_ORG_NAME = constants.GH_ORG_NAME
+    GL_ORG_NAME = constants.GL_ORG_NAME
+
+    def __init__(self, bot, name=None):
+        super().__init__(bot, name)
+
+        teams = dict()
+        try:
+            gh = github3.login(os.environ.get('GH_TOKEN'))
+            assert gh is not None
+            org = gh.organization(self.GH_ORG_NAME)
```

Comment:
You *could* move this into an `else` clause, e.g.:

```python
try:
    gh = github3.login(os.environ.get('GH_TOKEN'))
    assert gh is not None
except AssertionError:
    self.log.error('Cannot create github object, please check GH_TOKEN')
else:
    org = gh.organization(self.GH_ORG_NAME)
    for team in org.iter_teams():
        teams[team.name] = team
```

From python [docs](https://docs.python.org/3/tutorial/errors.html#handling-exceptions):

> The use of the else clause is better than adding additional code to the try clause because it avoids accidentally catching an exception that wasn’t raised by the code being protected by the try ... except statement.